webapp/channels/src/components/widgets/settings/text_setting.tsx

```diff
@@ -33,6 +34,12 @@ function TextSetting(props: Props) {
     function handleChange(event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) {
         if (props.type === 'number') {
             props.onChange(props.id, parseInt(event.target.value, 10));
+        } else if (props.type === 'text' && props.multiple) {
+            if (event.target.value === '') {
+                props.onChange(props.id, []);
+            } else {
+                props.onChange(props.id, event.target.value.split(','));
```

Comment:
Will you need to do any triming here? or is it done somewhere else? (I expect a space to be after the comma in most cases)

What about filtering empty values? (someone puts two commas together)

Reply:
The backend will reject and display an error for both the cases you mention. We can add some extra trimming on the front-end if necessary but for now I think we're ok